config.toml

```diff
@@ -144,26 +145,34 @@ summaryLength = 70
     posts = "/posts/:year/:month/:day/:filename/"
 
 [params.topbar]
-    enable = false
+    enable = true
+
+[params.topbar.services]
+    links = [
+        { name = "Status", url = "https://status.sonix.network" },
+        { name = "Metrics", url = "https://metric.sonix.network" },
+        { name = "PeeringDB", url = "https://peeringdb.com/ix/3607" },
+        { name = "IXP Manager", url = "https://manager.sonix.network/" }
+    ]
 
 [params.footer]
-about_us = """<p>Sonix is a community-driven non-profit Internet Exchange Point (IXP) dedicated to building a better-connected Sweden. Have an idea how we can help? We'd love to hear from you!</p>"""
+about_us = """<p>Sonix is a community-driven non-profit Internet Exchange Point (IXP) dedicated to building a better-connected Sweden.</p>"""
```

Comment:
We like to write SONIX with capital letters.

Reply:
I do use sonix with capital letters where I think it fits. My belief is that when you're highlighting the project, a concept or the brand itself, yes it should be in capital letters. But when writing general content it makes it harder to get fluidity in the text. In the footer itself, my personal opinion is that it looks good without capitalization, especially now since I added the SONIX-logo on top of it. This concept have been implemented overall on the site where I've made changes I believe.

It's an easy fix if you guys are set on this in stone.

Reply:
SONIX is an acronym and a proper noun. As far as I know it is supposed to be capitalized in English. We are also missing the fully typed out "Sweden Open Network Internet eXchange".

Reply:
Let's discuss this tomorrow as well so we can find a good spot for it. Something that is still missing is a proper about us page, which I do think is needed. This would be a great place to give the story behind the acronym and the team. I also think that we might need to come up with a few ways to live up to the new (maybe rather just now pointed out) transparency goal.